apps/web/src/design-system/table/Table.tsx

```diff
@@ -10,13 +10,23 @@ export interface ITableProps extends JSX.ElementChildrenAttribute {
   columns?: ColumnWithStrictAccessor<Data>[];
   data?: Data[];
   loading?: boolean;
+  pagination?: any;
```

Comment:
Is it possible to create a static type for this? { pageSize: number, total: number, etc...)